packages/experiments/src/components/Tile/examples/Tile.Media.Example.tsx

```diff
@@ -47,18 +107,7 @@ const ImageTile: React.StatelessComponent<IImageTileProps> = (props: IImageTileP
     <Tile
       contentSize={props.tileSize}
       itemName={<SignalField before={<NewSignal />}>{props.item.name}</SignalField>}
-      itemActivity={
```

Comment:
Why are you removing these? These currently represent the design for media-type items in normal Files view.

Reply:
This will place every line inside the `<SignalField/>`, which awkwardly indents all lines and centers the signal icons. I've moved the `SignalField` to where `itemActivity` is assigned above.